### ISSUE TITLE
[Attack Power Missing] Hunters and Rogues are 10AP short

### ISSUE THREAD

**Issue by arestavo** [dalaranwow/dalaran-wow#5747]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Hunters and rogues are missing 10 AP from agility, apparently this happens with a naked character and not just a geared character. 1 agility should equal 1 attack power, it currently is 10 attack power short.
**Expected behaviour**: Tell us what should happen instead
1 agility should equal 1 attack power for hunters and rogues. It currently is 10 attack power short.
**Steps to reproduce the problem**:

1.  Create a hunter or rogue.
2. See that 1 agility =/= 1 attack power because you are 10 attack power short.
3. 
**Include proofs for this behaviour**
![AP AGI bug](https://user-images.githubusercontent.com/50332138/59979622-8ed65900-95a7-11e9-809f-c19500190979.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Chenocide** [comment]
Would like to see you do some more testing with orcs and trolls, cause it's most like a problem with base stats of one or more races. Orc shamans are currently missing some Int too.

**Comment by Donlod** [comment]
This should be intended. There is a flat ap penalty of -10 or -20AP from agi/str on your AP/RAP. I dont exactly know how high the penalty is for the different classes. As for casters they should always have a -20 base AP penalty from str. So for a warlock its not meleeAP = str * 2 but AP = str * 2 - 20
